guides/common/modules/proc_promoting-a-content-view.adoc

```diff
@@ -51,5 +51,24 @@ Now the repository for the Content View appears in all environments.
 ----
 +
 Now the database content is available in all environments.
-
++
 To register a host to your Content View, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.
```

Comment:
Now that it's in between the two sections, this no longer makes sense here. Should move it to after both sections.